openpype/hosts/houdini/plugins/load/load_fbx.py

```diff
@@ -0,0 +1,171 @@
+# -*- coding: utf-8 -*-
+"""Fbx Loader for houdini.
+
+It's almost a copy of
+'load_bgeo.py'and 'load_alembic.py'
+however this one includes extra comments for demonstration.
+
+This plugin is part of publish process guide.
+"""
+import os
+
+from openpype.pipeline import (
+    load,
+    get_representation_path,
+)
+from openpype.hosts.houdini.api import pipeline
+
+
+class FbxLoader(load.LoaderPlugin):
+    """Load fbx files to Houdini."""
+
+    label = "Load FBX"
+    families = ["staticMesh", "fbx"]
+    representations = ["fbx"]
+
+    # Usually you will use these value as default
+    order = -10
+    icon = "code-fork"
+    color = "orange"
+
+    def load(self, context, name=None, namespace=None, data=None):
+
+        # get file path
+        file_path = self.get_file_path(context)
+
+        # get necessary data
+        namespace, node_name = self.get_node_name(context, name, namespace)
+
+        # create load tree
+        nodes = self.create_load_node_tree(file_path, node_name, name)
+
+        self[:] = nodes
+
+        # Call containerise function which does some
+        # automations for you
+        containerised_nodes = self.get_containerised_nodes(
+            nodes, context, node_name, namespace
+        )
+
+        return containerised_nodes
```

Comment:
You may consider to return pipeline.containerise function here instead of having an extra get_containerised_node function to call "pipeline.containerise" function. You can take reference from other loaders.

Reply:
You are totally right. 
I did it this way to make it easier to understand because this whole PR is part of "publish process development guide" that we also want it to work for production.

Reply:
> You are totally right. I did it this way to make it easier to understand because this whole PR is part of "publish process development guide" that we also want it to work for production.
> 
> please tell me if you think I don't have to do this and it's already easy to read

In my opinion, I think it is better to stick with using pipeline.containerise for alignment with the codes. You can find similar containerise functions in other Hosts like Maya, Max and Nuke.